patroni/config.py

```diff
@@ -45,6 +45,7 @@ class Config(object):
         'master_start_timeout': 300,
         'synchronous_mode': False,
         'synchronous_mode_strict': False,
+        'standby_cluster': '',
```

Comment:
I've though a little bit about format of `standby_cluster` configuration and would suggest to follow postgres naming conventions here: https://www.postgresql.org/docs/10/static/recovery-config.html
```python
{
    'host': 'localhost',
    'port': 12345,
    'primary_slot_name': 'foo-bar',
    'restore_command': 'true',
    'archive_cleanup_command': ''
    'recovery_min_apply_delay': '10min'
}
```

We can assume that this is a standby cluster when either `host` or `restore_command` are defined.